DataProducts/test/lassipatanen/Job/JobPosting.json

```diff
@@ -0,0 +1,201 @@
+                "$ref": "#/components/schemas/JobPostingRequest"
+              }
+            }
+          }
+        },
+        "description": "Job Posting"
+      },
+      "parameters": []
+    }
+  },
+  "components": {
+    "schemas": {
+      "HTTPValidationError": {
+        "title": "HTTPValidationError",
+        "type": "object",
+        "properties": {
+          "detail": {
+            "title": "Detail",
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ValidationError"
+            }
+          }
+        }
+      },
+      "ValidationError": {
+        "title": "ValidationError",
+        "required": [
+          "loc",
+          "msg",
+          "type"
+        ],
+        "type": "object",
+        "properties": {
+          "loc": {
+            "title": "Location",
+            "type": "array",
+            "items": {
+              "anyOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "integer"
+                }
+              ]
+            }
+          },
+          "msg": {
+            "title": "Message",
+            "type": "string"
+          },
+          "type": {
+            "title": "Error Type",
+            "type": "string"
+          }
+        }
+      },
+      "JobPostingRequest": {
+        "title": "JobPostingRequest",
+        "type": "object",
+        "description": "",
+        "properties": {
+          "keywords": {
+            "type": "string",
+            "description": "Comma separated list of keywords to search for."
+          }
+        }
+      },
+      "JobPostingResponse": {
+        "type": "object",
+        "x-examples": {
+          "example-1": {
+            "employer": "",
+            "location": {
+              "city": ""
+            },
+            "basicInfo": {
+              "title": "",
+              "description": ""
+            },
+            "published_at": ""
+          }
+        },
+        "properties": {
+          "employer": {
+            "type": "string"
+          },
+          "location": {
+            "type": "object",
+            "required": [
+              "city"
+            ],
+            "properties": {
+              "city": {
+                "type": "string"
+              }
+            }
+          },
+          "basicInfo": {
+            "type": "object",
+            "required": [
+              "title",
+              "description"
+            ],
+            "properties": {
+              "title": {
+                "type": "string"
+              },
+              "description": {
+                "type": "string"
+              }
+            }
+          },
+          "published_at": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "employer",
+          "location",
+          "basicInfo",
+          "published_at"
+        ]
+      }
+    }
+  }
+}
```

Comment:
The pre-commit hooks complain about the file not having a trailing newline; you could just run the pre-commit hooks to let them fix it.